Modules/ZDC/src/ZDCRawDataTask.cxx

```diff
@@ -952,12 +980,49 @@ bool ZDCRawDataTask::addNewHisto(std::string type, std::string name, std::string
         return true;
       }
     }
+
+
+    if (type == "COUNTSA") {
+      // Check if Histogram Exist
+      if (std::find(fNameHisto.begin(), fNameHisto.end(), name) == fNameHisto.end()) {
+        fNameHisto.push_back(name);
+        h1d.histo = new TH1F(hname, htit, fNumBinX, fMinBinX, fMaxBinX);
+        h1d.condHisto.push_back(condition);
+        ih = (int)fMatrixHistoCounts_a[mod][ch].size();
+        fMatrixHistoCounts_a[mod][ch].push_back(h1d);
+
+        if (ih < (int)fMatrixHistoCounts_a[mod][ch].size()) {
+          getObjectsManager()->startPublishing(fMatrixHistoCounts_a[mod][ch].at(ih).histo);
+          try {
+            getObjectsManager()->addMetadata(fMatrixHistoCounts_a[mod][ch].at(ih).histo->GetName(), fMatrixHistoCounts_a[mod][ch].at(ih).histo->GetName(), "34");
```

Comment:
what is this ?

Reply:
It is the same code used in line 965.
We want to use the histograms of type COUTSA to generate a trending plot of the rate during a run.
The difference between COUTS and COUTSA is the the second want is zeroed at each cycle and thus provides an estimate of the  instantaneous rate and not the average one.